### PR TITLE
PROD-773 Fatar error when Clearing default data from BuddyBoss Tools

### LINKED ISSUE #2749: Fatar error when Clearing default data from BuddyBoss Tools
**Describe the bug**

When we have data related to Forum discussions and the component Forum discussions is inactive, clearing default data from BuddyBoss > Tools will raise a fatal error

**To Reproduce**
=> Activate Forum discussions component first
=> Import data related to Forum discussions like Discussions/Replies
=> Deactivate Forum discussions component
=> The again go to BuddyBoss > Tools > Click on Clear default data
=> The Fatal Error will appear

**Expected behavior**
The fatal error should not arise when the Forum discussions component is inactive. . 

**Screenshots**
https://prnt.sc/1l2vo3b

Video: 
https://www.loom.com/share/96a20b76c3ab4ebc9a0d342a7311d48b

**Environment**
On Local MAMP Environment

**Jira issue** : [PROD-773]

[PROD-773]: https://buddyboss.atlassian.net/browse/PROD-773?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### PR DESCRIPTION
**Pull request by Nayeem-BB**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #2749  .

### How to test the changes in this Pull Request:

=> Activate Forum discussions component first
=> Import data related to Forum discussions like Discussions/Replies
=> Deactivate Forum discussions component
=> The again go to BuddyBoss > Tools > Click on Clear default data
=> The Fatal Error should not appear anymore in this fix as stated in the issue

### Proof Screenshots or Video

https://www.loom.com/share/609d53c82adf4fa391844b16006815cb

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-core/bp-core-tools-default-data.php [hunk 1]
```diff
@@ -243,11 +243,13 @@ function bp_dd_delete_dummy_forum() {
 	/**
 	 * Delete Forums
 	 */
+	if ( bp_is_active( 'forums' ) ) {
 	$forums = bp_get_option( 'bp_dd_imported_forum_ids' );
 	if ( ! empty( $forums ) ) {
 		foreach ( (array) $forums as $forum_id ) {
 			wp_delete_post( $forum_id );
 			bbp_delete_forum( $forum_id );
+			}
```
Comment (chetansatasiya) [inline src/bp-core/bp-core-tools-default-data.php:252]: @Nayeem-BB Need the proper tabing here.

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
@Nayeem-BB Thanks for the PR but we need to follow the proper tabing and spacing as per coding standard.

File: src/bp-core/bp-core-tools-default-data.php (hunk 1 above)
Comment (Nayeem-BB) [reply src/bp-core/bp-core-tools-default-data.php]: Done @chetansatasiya  Thank you 🙂

**Review by jitendrabanjara1991 — APPROVED** [review]
LGTM

**Pull request merged**